test/unit/transports/local_test.rb

```diff
@@ -93,4 +94,36 @@ def mock_run_cmd(cmd, &block)
       end
     end
   end
+
+  describe 'when running on Windows' do
+    let(:connection) { TransportHelper.new(:windows).transport.connection }
+
```

Comment:
Remove extra newline - it's more common to group all the `let`s together than space them out.